.github/workflows/kubernetes-install.yaml

```diff
@@ -0,0 +1,59 @@
+name: Kubernetes install
```

Comment:
Is this similar to what I was doing here @psschwei ? https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/.github/actions/helm-lint/action.yml#L36

Lines are commented because it was giving me error the test and I hadn't the time yet to investigate it.

Reply:
looks similar, yes

Reply:
I don't know your opinion but I think we could continue in that action instead to create a new one (with your logic obviously, mine was not working haha)

Reply:
I'd lean towards keeping them separate -- yours seems to be around testing helm functionality, whereas this one is aiming to check that we can run the sample notebooks on kubernetes

Reply:
Mmm... Probably then can be separated:
- lint
- installation
- testing with notebooks

Reply:
Both can be actions and reused in workflows like the helm-verify or the release one.

Reply:
will tackle splitting things up if/when I can get this working :pray: